src/dmclock_client.h

```diff
@@ -122,9 +122,12 @@ namespace crimson {
 	} else {
 	  Counter delta =
 	    delta_counter - it->second.delta_prev_req;
+#ifdef USE_SEPARATED_RHO_CAL
```

Comment:
This removes subtracting my_rho/my_delta that induce a vicious cycle.
In random mode, many requests can go one specific server sometimes by random manner.
The vicious cycle is as follows.
1. many requests go to one specific server
2. many acks are returned for the server
3. my_rho/my_delta value for the server increase
4. rho/delta value for the server is decreased by subtraction of large my_rho/my_delta
5. the server receive small rho/delta
6. the server services more requests from the client because of the client's small rho/delta value
7. go to step 2 again

As a result, there will be bias for rho/delta value between clients.